project-base/storefront/public/locales/sk/common.json

```diff
@@ -159,6 +159,7 @@
     "Item {{ itemName }} has been sold out.": "Item {{ itemName }} has been sold out.",
     "Last name": "Priezvisko",
     "Last name must be at most {{ max }} characters": "",
+    "Last visited products": "",
```

Comment:
Maybe we can fill also this translation?

Reply:
A lot of translations is missing, but filled.